src/com/palmergames/bukkit/towny/db/TownySQLSource.java

```diff
@@ -1644,7 +1658,26 @@ public synchronized boolean saveTown(Town town) {
         return false;
     }
 
-    @Override
+	@Override
+	public synchronized boolean savePlotGroup(PlotObjectGroup group) {
```

Comment:
Like the flatfile, this is missing a number of things like resident, type, perm line.